corehq/apps/sms/models.py

```diff
@@ -881,12 +881,18 @@ class MessagingEvent(models.Model, MessagingStatusMixin):
     STATUS_COMPLETED = 'CMP'
     STATUS_NOT_COMPLETED = 'NOT'
     STATUS_ERROR = 'ERR'
+    STATUS_EMAIL_BOUNCED = 'BNC'
+    STATUS_EMAIL_SENT = 'SND'
+    STATUS_EMAIL_DELIVERED = 'BNC'
```

Comment:
Is this intentionally the same value as `STATUS_EMAIL_BOUNCED`?

Reply:
Ooops, good catch.